nurseconnect/constants.py

```diff
@@ -0,0 +1,10 @@
+from django.utils.translation import ugettext_lazy as _
+
+MALE = "m"
+FEMALE = "f"
+UNSPECIFIED = "-"
+GENDERS = {
```

Comment:
If you can consider rather making this `SEX`, its not subjective and cascade that change.  If the scope of work allows, naturally.